.pre-commit-config.yaml

```diff
@@ -19,3 +19,23 @@ repos:
       args: ['-b master']
```

Comment:
(not this PR, but you probably want `args: [-b, master]` -- I'm actually surprised the current invocation isn't an error, but I guess it's parsing it the same as `-b= master` (so it would prevent commits to a branch name that starts with a space)

Reply:
Agree.
It actually does not need `args`, the current hook already have `master` as a default branch.